mne/io/egi/egimff.py

```diff
@@ -887,16 +891,21 @@ def _read_evoked_mff(fname, condition, channel_naming='E%d', verbose=None):
     # Add EEG reference to info
     # Initialize 'custom_ref_applied' to False
     info['custom_ref_applied'] = False
-    with mff.directory.filepointer('history') as fp:
-        history = mffpy.XML.from_file(fp)
-    for entry in history.entries:
-        if entry['method'] == 'Montage Operations Tool':
-            if 'Average Reference' in entry['settings']:
-                # Average reference has been applied
-                projector, info = setup_proj(info)
-            else:
-                # Custom reference has been applied that is not an average
-                info['custom_ref_applied'] = True
+    try:
+        fp = mff.directory.filepointer('history')
+    except ValueError:  # should probably be FileNotFoundError upstream...
```

Comment:
FYI @ephathaway I had to add this, with the code before writing with `history=None` raised an error when the resulting file was read with `read_evokeds_mff`. This catches the error that the history file is not written. Upstream in mffpy this should probably be a FileNotFoundError rather than ValueError. I can open a PR for this in mffpy if you agree

Reply:
That would be great. Thanks! And yes, I think we should treat history.xml as optional.

Reply:
https://github.com/BEL-Public/mffpy/pull/89

Pushed a commit to make this PR compatible with ValueError (backward compat) and FileNotFoundError (future compat)